lunch-application/src/theme/Home.vue

```diff
@@ -61,23 +61,35 @@
         .then(response => {
           // JSON responses are automatically parsed.
           console.log(response)
-          this.posts = response.data
+          this.restaurantData = response.data
+          return this.restaurantData
         })
         .catch(e => {
           this.errors.push(e)
         })
+    },
+
+    methods: {
+      lunchAlert: function () {
+        var random = Math.floor(Math.random() * 5)
+        VueOnToast.ToastService.pop('success', 'test', 'lunch today is at: ' + this.restaurantData[random].restaurantName)
+      }
+
     }
   }
 </script>
-
 <style scoped>
   .options{
-    width: 40%;
-    height:100%;
+    order: 1;
+    flex-basis: 40%;
+    height: 300px;
```

Comment:
you can pull these out from here and .topFive so they aren't repeated (and there's only one place to update if you have to change them)